bin/update_bitmasks

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env python
+
+"""
+Update doc/bitmasks.rst based upon code descriptions
+"""
+
+bitmask_template = """
+=================
+Bit Masks in DESI
+=================
+
+.. NOTE: this file is generated by desidatamodel/bin/update_bitmasks, e.g.
+..   cd desidatamodel/doc
+..   python ../bin/update_bitmasks > bitmasks.rst && make html
+
+This page describes the bitmasks found in DESI files.
+For details on working with these values, please see the tutorial_ on
+that topic.
+
+Redshift Fitting (Redrock) Masks
+--------------------------------
+
+The **ZWARN** bitmask in redshift catalogs indicates known problems with a
+particular redshift fit or associated QA.
+ZWARN==0 is good; any non-zero value indicates a potential problem.
+This mask will be described in more detail in the Redrock paper
+(Bailey et al 2023 in prep), as well as Section 5.3.1 of the
```

Comment:
You might want to mark this as TODO as well.

Reply:
I didn't list this as "TODO" since the Redrock paper will remain "in prep" when we go live for EDR, so the text as-is is correct for the purposes of EDR.

Reply:
Ah, that's OK, I thought that was the EDR paper.